nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/admin/SetHealthCheckServiceConfiguration.java

```diff
@@ -0,0 +1,402 @@
+            }
+            report.setActionExitCode(ActionReport.ExitCode.SUCCESS);
+        }
+        catch (TransactionFailure ex) {
+            logger.log(Level.WARNING, "Exception during command ", ex);
+            report.setMessage(ex.getCause().getMessage());
+            report.setActionExitCode(ActionReport.ExitCode.FAILURE);
+        }
+        if (dynamic && (!server.isDas() || targetConfig.isDas())) {
+            configureDynamically(service, config.getCheckerByType(checkerType));
+        }
+    }
+
+    private <C extends Checker, O extends HealthCheckExecutionOptions> void configureDynamically(
+            BaseHealthCheck<O, C> service, C config) {
+        if (service.getOptions() == null) {
+            service.setOptions(service.constructOptions(config));
+        }
+        healthCheckService.registerCheck(config.getName(), service);
+        healthCheckService.reboot();
+        if (service instanceof BaseThresholdHealthCheck) {
+            configureDynamically((BaseThresholdHealthCheck<?, ?>) service);
+        }
+    }
+
+    private void configureDynamically(BaseThresholdHealthCheck<?, ?> service) {
+        if (thresholdCritical != null) {
+            service.getOptions().setThresholdCritical(Integer.valueOf(thresholdCritical));
+            report.appendMessage(strings.getLocalString(
+                    "healthcheck.service.configure.threshold.critical.success",
+                    "Critical threshold for {0} service is set with value {1}.", serviceName, thresholdCritical));
+            report.appendMessage("\n");
+        }
+        if (thresholdWarning != null) {
+            service.getOptions().setThresholdWarning(Integer.valueOf(thresholdWarning));
+            report.appendMessage(strings.getLocalString("healthcheck.service.configure.threshold.warning.success",
+                    "Warning threshold for {0} service is set with value {1}.", serviceName, thresholdWarning));
+            report.appendMessage("\n");
+        }
+        if (thresholdGood != null) {
+            service.getOptions().setThresholdGood(Integer.valueOf(thresholdGood));
+            report.appendMessage(strings.getLocalString("healthcheck.service.configure.threshold.good.success",
+                    "Good threshold for {0} service is set with value {1}.", serviceName, thresholdGood));
+            report.appendMessage("\n");
+        }
+    }
+
+    private <C extends Checker> Checker updateProperties(Checker config, Class<C> type) throws PropertyVetoException {
+        updateProperty(config, "enabled", config.getEnabled(), String.valueOf(enabled), Checker::setEnabled);
+        updateProperty(config, "time", config.getTime(), time, Checker::setTime);
+        updateProperty(config, "time-unit", config.getUnit(), unit, Checker::setUnit);
+        if (HoggingThreadsChecker.class.isAssignableFrom(type)) {
+            HoggingThreadsChecker htConfig = (HoggingThreadsChecker) config;
```

Comment:
`htConfig` doesn't confirm to our code style.
It should be something more like `hoggingThreadsConfig`.